runtime/src/bank.rs

```diff
@@ -1548,7 +1548,8 @@ impl Bank {
                     if (self
                         .feature_set
                         .is_active(&feature_set::timestamp_bounding::id())
-                        && self.ancestors.contains_key(&timestamp_slot))
+                        && self.slot().checked_sub(timestamp_slot)?
+                            <= self.epoch_schedule().slots_per_epoch)
```

Comment:
@aeyakovenko , I think that the vote accounts with stake beyond 99.5% will get swept up in the stake-weighted median calculation, so I've limited this to just filtering out anything older than an epoch.

Reply:
good enough for now